package.json

```diff
@@ -27,10 +27,22 @@
     "url": "https://github.com/strongloop/loopback-soap/issues"
   },
   "devDependencies": {
+    "chai": "^3.2.0",
     "eslint": "^3.12.2",
-    "eslint-config-loopback": "^7.0.0"
+    "eslint-config-loopback": "^8.0.0",
+    "loopback": "^3.0.0",
+    "mocha": "^3.2.0"
   },
-  "license": "UNLICENSED",
+  "license": "MIT",
   "dependencies": {
+    "async": "^2.1.5",
+    "curl": "^0.1.4",
+    "debug": "^2.2.0",
+    "ejs": "^2.5.5",
+    "lodash": "^4.17.4",
+    "spawn-sync": "1.0.15",
+    "strong-globalize": "^2.6.0",
+    "strong-soap": "^1.0.1",
+    "underscore.string": "^3.3.4"
```

Comment:
"underscore.string": "^3.3.4" used in generator-soap -->      op.operationId = _s.camelize(
                op.operationId.split(/[\.\s\/-:\*]+/).join('_'));